### DISCUSSION TITLE
Chai Blockchain 2.1.1 stuck on starting services Full Node

### DISCUSSION THREAD

**Discussion by jlaboy71** [Chia-Network/chia-blockchain#16829] [Support]
I have uninstalled and reinstalled the app. I have gone from version 2.1.1 to 2.1.0, I have uninstalled and renamed the c:/program files/chia director to chia-old and reinstalled the new version again. I have also made sure that windows have the latest updates installed. It has been like this for a while. Is there anything I can do to see if this is working as expected by looking at log file or some other way to see that it is processing. Thanks

**Comment by OverActiveBladderSystem** [comment]
This sounds like the "You need to upgrade your blockchain V1 database to a blockchain V2 database" issue some are experiencing because Chia version 2.1.0 and above no longer support the older V1 database.

Below is a copy/paste of the help message from the [Chia Discord](https://discord.gg/chia) support channel. Side note, the database should be located under your username directory in windows, its within the db folder and you will see if it has the V1 within its name.
`c:\Users\%USERNAME%\.chia\mainnet\db`

The blockchain database v2 is required for chia versions 2.1 and newer.
Database upgrade steps either via cli or torrent snapshot download:

Steps for the CLI command:
1. Stop the chia client (this can be done while chia is running but stopping the client saves from any potential issues)
2. Run the command `chia db upgrade` (you do not need the input or output paths if these are still default)
- This process will take some time and requires additional space as the v1 db is compacted into the v2 db which leaves the v1 db intact
3. Verify the config file (\.chia\mainnet\config\config.yaml) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
4. After the above completes start the chia client
5. If all loads up properly (might take 5-10 minutes to load) then you can safely delete the v1 db found here (\.chia\mainnet\db\blockchain_v1_mainnet.sqlite)


Steps for using the torrent: 
1.Download the torrent file from https://www.chia.net/downloads/#database-checkpoint
2. Use a torrent client to download the full db
3. Unpack/reassemble the torrent file that was downloaded (on windows I used 7zip, Mac and linux have built in tools that work for this)
4. Move the db to the correct folder (\.chia\mainnet\db\) and update the name to "blockchain_v2_mainnet.sqlite" (removing the date information in the name)
5. Verify the config file (\.chia\mainnet\config\config.yaml) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
6. Launch chia and wait for a bit (the height to hash and peers files need to be built so this can take 5-10 minutes)

**Comment by BrandtH22** [comment]
Hey @jlaboy71 , as a heads up we can generally provide more timely, thorough support on our discord server (https://discord.gg/chia).

As OverActiveBladderSystem mentioned, it sounds like this is an issue with the database version. (Thank you OverActiveBladderSystem!!!)
Did following the provided steps resolve the issue with starting the node?

**Comment by jlaboy71** [comment]
Thank you guys. Right before chatGPT went under I was able to create a GPT
and tell it to become a Chia Guru providing various Chia related websites.
Then by asking it tech questions it took me to the page you referenced.
Thanks for pointing that out. Not sure if this was something that was made
known and I missed it but if it wasn’t it killed some time and provided
some frustration for software that typically is a set it and forget it.

Thanks!

**Comment by BrandtH22** [comment]
Hey @jlaboy71 , thank you for the feedback and confirmation that everything is up and running.
There have been notes about the v1 db being deprecated for the past couple of releases but many users seem to have missed those.

I am going to close the ticket for now but don't hesitate to jump into our discord or open a new ticket if other issues or questions arise
